test/activation_based/test_op_counter_neuromc_runtime.py

```
@@ -0,0 +1,339 @@
+        w,
+        None,
+        [1, 1],
+        [1, 1],
+        [1, 1],
+        False,
+        [0, 0],
+        1,
+        output_mask,
+    )
+
+    reads, writes = _access_convolution_backward(args, {}, out)
```

Ruff (0.15.12)

[warning] Unpacked variable `reads` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)